### PR TITLE
Application header URL change

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/RHTAPBUGS-144

## Description

Application header update

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Screenshot 2023-05-11 at 7 22 32 PM
Screenshot 2023-05-11 at 7 24 06 PM
Screenshot 2023-05-11 at 7 24 28 PM

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by christianvogt** [comment]
@abhinandan13jan in the screenshot I would expect the external link icon to be on the same line as the URL.

**Comment by abhinandan13jan** [comment]
@christianvogt updated
Screenshot 2023-05-10 at 8 43 33 PM
Screenshot 2023-05-10 at 8 44 25 PM

**Comment by christianvogt** [comment]
The truncation works as expected. Not sure how many trailing characters UX wants to see but the current default seems low because you don't get to see much of the end of the URL.
We can probably do `Math.min(url.length / 2, 20)`.

File: src/components/ApplicationDetails/ApplicationHeader.tsx [hunk 1]
```diff
@@ -54,25 +43,30 @@ export const ApplicationHeader: React.FC<{ application: ApplicationKind }> = ({
...
-                <Button
-                  variant="plain"
-                  data-testid="route-copy-icon"
-                  isInline
-                  onClick={onCopyButtonClick}
-                >
-                  <OutlinedCopyIcon color={linkColor.value} />
-                </Button>
+          {selectedComponentRoute &&
+            (selectedComponentRoute.length > 40 ? (
```
Comment (christianvogt) [inline src/components/ApplicationDetails/ApplicationHeader.tsx:47]: `Truncate` will only truncate if necessary. Therefore I think we can always render this path without the conditional rendering.

File: src/components/ApplicationDetails/ApplicationHeader.tsx [hunk 2]
```diff
@@ -54,25 +43,30 @@ export const ApplicationHeader: React.FC<{ application: ApplicationKind }> = ({
...
+              <Tooltip content={selectedComponentRoute} data-test-id="component-route-tooltip">
+                <ExternalLink href={selectedComponentRoute} dataTestID="component-route-link">
```
Comment (christianvogt) [inline src/components/ApplicationDetails/ApplicationHeader.tsx:49]: Why two different data test ID attributes?

**Comment by abhinandan13jan** [comment]
@christianvogt updated, I've set the trailing truncate position to min(half length,30) and the tooltip and conditional issue

Screenshot 2023-05-11 at 7 22 32 PM
Screenshot 2023-05-11 at 7 24 06 PM
Screenshot 2023-05-11 at 7 24 28 PM

**Pull request merged**